### PR TITLE
add validation to user_event create

### PR DESCRIPTION
**Pull request by Zenjjim**
# PR NAME
## Description
Link to Trello task: ...

Comments/issues:

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: app/content/views/user_event.py [hunk 1]
```diff
@@ -53,13 +54,18 @@ def create(self, request, event_id):
         """ Registers a user with the specified event_id and user_id """
         try:
             event = Event.objects.get(pk=event_id)
-            user = User.objects.get(user_id=request.id)
+            if event.start_registration_at > today():
+                return Response({'detail': 'The registration for this event has not started yet.'}, status=400)
+            if event.end_registration_at < today():
+                return Response({'detail': 'The registration for this event has ended.'}, status=400)
+            if event.sign_off_deadline < today():
+                return Response({'detail': 'Sign off deadline cannot be after deadline.'}, status=400)
```
Comment (Eirsteir) [inline app/content/views/user_event.py]: Bra, men sjekker vi ikke dette på modellen? Det er uansett god praksis å ha logikken så lavt/nærme dataen som mulig, nemlig modellen.

Comment (Zenjjim) [reply app/content/views/user_event.py]: Det går ikke ann å teste i modellen, får ikke feilmeldingne om jeg ikke har den i create

Comment (Zenjjim) [reply app/content/views/user_event.py]: Det er derfor olaf hadde buggen sin

Comment (Eirsteir) [reply app/content/views/user_event.py]: Okei, vi får se nærmere på det senere.

**Pull request merged**